### DISCUSSION TITLE
bookmarks and tabs in one panel

### DISCUSSION THREAD

**Discussion by j0lol** [mbnuqw/sidebery#859] [Ideas]
hi! ive just started using sidebery and i love how configurable it is. on v5 i noticed there was bookmark panels? ive been using pinned tabs as "bookmarks" with an auto unloader extension to stop that eating my ram, but I was wondering if it was possible to merge them into one panel? it would make my use case a lot less hacky :p

basically what i want is a small row of bookmarks with just favicons above tabs :)

**Comment by j0lol** [comment]
wow, the 'Additional "New tab" buttons' option is exactly what I wanted! sidebery truly is infinitely customizable!! 💕

**Reply by weekendweekender** [reply]
hey did you find a solution?

**Reply by j0lol** [reply]
i dont think so sorry
